docs/collateral.md

```diff
@@ -109,96 +113,133 @@ interface ICollateral is IAsset {
+Some defi protocols yield returns by increasing the token balances of users, called _rebasing_. For instance, ATokens from Aave and stETH from Lido are both rebasing tokens. While people often like this, smart contracts certainly do not. 
 
-In general any rebasing token can be wrapped to be turned into an appreciating exchange rate token, and vice versa. It's even possible to split the difference, if you want. But for a token to be used in the Reserve protocol as collateral, it's important that _all_ rebasing be eliminated from the token.
+The Reserve Protocol cannot directly hold rebasing tokens. However, the protocol can indirectly hold a rebasing token, if it's wrapped by another token that does not itself rebase, but instead appreciates only through exchange-rate increases. Any rebasing token can be wrapped to be turned into an appreciating exchange-rate token, and vice versa.
 
-To use a rebasing token as collateral backing, the ERC20 needs to be replaced with an ERC20 that is non-rebasing. This is _not_ a change to the collateral plugin contract itself. Instead, the collateral plugin designer needs to provide a wrapping ERC20 contract that RToken issuers/redeemers will have to deposit/withdraw into when they perform issuance/redemption. In the future this transformation should be provided as a type of zap, but at the time of this writing everything is still manual.
+To use a rebasing token as collateral backing, the rebasing ERC20 needs to be replaced with an ERC20 that is non-rebasing. This is _not_ a change to the collateral plugin contract itself. Instead, the collateral plugin designer needs to provide a wrapping ERC20 contract that RToken issuers or redeemers will have to deposit into or withdraw from. We expect to automate these transformations as zaps in the future, but at the time of this writing everything is still manual.
 
-For an example of what a token wrapper that performs this transformation looks like, check out [contracts/plugins/aave/StaticATokenLM.sol](../contracts/plugins/aave/StaticATokenLM.sol). This is a standard wrapper used by many protocols to wrap Aave ATokens into StaticATokens. A thinned-down version of this contract makes a good starting point for developing other ERC20 wrappers. But if the defi protocol is well-integrated into defi, it's likely there already exists a wrapping token contract that can be vendored and used directly.
+For an example of a token wrapper that performs this transformation, see [StaticATokenLM.sol](../contracts/plugins/aave/StaticATokenLM.sol). This is a standard wrapper to wrap Aave ATokens into StaticATokens. A thinned-down version of this contract makes a good starting point for developing other ERC20 wrappers -- but if the token is well-integrated in defi, a wrapping contract probably already exists.
```

Comment:
Let's add a short paragraph below this saying that the same approach can be used to tokenize a non-tokenized/non-transferrable position.